R/makeNeighbourhoods.R

```diff
@@ -30,63 +32,104 @@
 #' neighbourhoods.
 #'
 #' @author
-#' Mike Morgan, refined sampling written by Emma Dann
+#' Emma Dann, Mike Morgan
 #'
 #' @examples
 #'
 #' requires(igraph)
 #' m <- matrix(rnorm(10000), ncol=10)
 #' milo <- buildGraph(m, d=10)
 #'
-#' milo <- makeNeighbourhoods(milo, prop=0.3)
+#' milo <- makeNeighbourhoods(milo, prop=0.1)
 #' milo
 #' @name makeNeighbourhoods
-NULL
-
-#' @export
```

Comment:
Please make sure that the correct attributes are retained, i.e. `@export`, `@rdname', etc